gabbi/case.py

```diff
@@ -419,7 +437,8 @@ def _test_data_to_string(self, data, content_type):
                 else:
                     return info
         else:
-            data = json.dumps(data)
+            data = self.replace_template(data, retype_numerals=True)
```

Comment:
How do we know that numeric values is what we want here? (Though I suppose it doesn't actually hurt either way.)